cargo-shuttle/src/args.rs

```diff
@@ -308,23 +311,29 @@ pub struct DeployArgs {
     /// WIP: Deploy this Docker image instead of building one
     #[arg(long, short = 'i', hide = true)]
     pub image: Option<String>,
-    /// Don't follow the deployment status, exit after the deployment begins
-    #[arg(long, visible_alias = "nf")]
-    pub no_follow: bool,
```

Comment:
Don't we want to keep this for the regular deploy command? Or am I reading this removal wrong?

Reply:
It got moved into tracking_args

Reply:
I see it now :see_no_evil: